pkg/pillar/hypervisor/xen.go

```diff
@@ -119,12 +119,6 @@ func (ctx xenContext) CreateDomConfig(domainName string, config types.DomainConf
 			config.VirtualizationMode)
 	}
 
-	if config.IsContainer {
-		kernel = "/hostfs/boot/kernel"
-		ramdisk = "/usr/lib/xen/boot/runx-initrd"
-		extra = extra + " root=9p-xen dhcp=1"
```

Comment:
Old kvm code and new common code has console=hvc0. Do we set that for Xen somewhere else?

Reply:
No -- we just rely on the fact that the linux kernel booted under Xen gets hvc0 as the default console by default. Under KVM hvc0 is only available if you explicitly ask for it.

Which means that console=hvc0 is basically a no-op for Xen, but it doesn't hurt -- and hence it is moved to a common code now.